### PR TITLE
Dropzone component + fixes

### PR DESCRIPTION
**Pull request by VeraZab**
closes: #333

### CODE REVIEW

**Comment by VeraZab** [comment]
ready for review cc @nicolaskruchten

File: src/components/widgets/Dropzone.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+      }
+    }
+  }
+
+  render() {
+    return (
+      <Drop
+        ref="dzone"
+        onDrop={this.onDrop}
+        disableClick={true}
```
Comment (nicolaskruchten) [inline src/components/widgets/Dropzone.js]: I would enable click, so that people can also use the file dialog instead of being required to drag.

Comment (nicolaskruchten) [reply src/components/widgets/Dropzone.js]: `/create` supports clicking on the word "upload" for this purpose, but really the whole area can be clickable and it works just as well, like I did here https://react-pivottable.js.org/

**Comment by VeraZab** [comment]
screen shot 2018-03-17 at 11 09 35 am

**Comment by nicolaskruchten** [comment]
Dancer!

**Pull request merged**